c/pthread-driver-races/LICENSE.txt

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I believe this should be a symlink (though it might actually be and github is just not telling me?)